release/release.go

```diff
@@ -1297,3 +1353,181 @@ func linuxRelease(v version.Version) {
 func canPerformStableRelease(v version.Version) bool {
 	return v.IsStable() && env.EvgIsTagTriggered()
 }
+
+func downloadMongodAndShell(v string) {
+	err := os.Mkdir("bin", 0700)
+	if err != nil && !os.IsExist(err) {
+		check(err, "create bin dir")
+	}
+
+	pf, err := platform.GetFromEnv()
+	check(err, "get platform")
+
+	feedURL := "http://downloads.mongodb.org/full.json"
+
+	var feed download.ServerJSONFeed
+
+	res, err := http.Get(feedURL)
+	check(err, "get the server JSON feed")
+
+	err = json.NewDecoder(res.Body).Decode(&feed)
+	check(err, "decode JSON feed")
+
+	url, githash, serverVersion, err := feed.FindURLHashAndVersion(v, pf.Name, string(pf.Arch), "enterprise")
+	check(err, "get URL from JSON feed")
+
+	fmt.Printf("URL: %v\n", url)
+	fmt.Printf("GitHash: %v\n", githash)
+	fmt.Printf("Version: %v\n", serverVersion)
+
+	downloadBinaries(url)
+	if semver.Compare(fmt.Sprintf("v%s", serverVersion), "v6.0.0") > -1 {
```

Comment:
```suggestion
	if semver.Compare(fmt.Sprintf("v%s", serverVersion), "v6.0.0") >= 0 {
```
nit: a little bit hard to read.

Reply:
Also good suggestion